### ISSUE TITLE
Obstacle Placement

### ISSUE THREAD

**Issue by ChristianBirchler** [ChristianBirchler/sdc-scissor#39]
- [x] Define interface @ChristianBirchler 
- [x] Add bumps @tanzilkm 
- [x] Add cylinder @tanzilkm @ChristianBirchler 
- [x] Refactoring @ChristianBirchler

**Comment by ChristianBirchler** [comment]
Hi @tanzilkm,

I added some basic representative objects/classes for bumps and delineators. I think it is best to follow now the KISS principle to make fast progress. In the end, I will do a refactoring by applying specialized Factory patterns since in the future these objects need also be instantiated depending on different simulators.

**Comment by tanzilkm** [comment]
Hi @ChristianBirchler ,
The Cylinder Obstacle and Bump were added. The interpolation method needs to be improved. Please review and let me know your feedback

**Comment by ChristianBirchler** [comment]
Thanks @tanzilkm, I will review it and do some refactorings. Can you please share with me the CLI commands you used to test the application with obstacles?

**Comment by ChristianBirchler** [comment]
@tanzilkm, I added another task. We should also write test cases. I think it is best to wait with them so that I can refactor first the code.

**Comment by ChristianBirchler** [comment]
@tanzilkm, I just figured out that there is no real need to write test cases for this issue (at least not now). I removed the task. I will apply the Abstract Factory Pattern for the creation of the obstacles since in the future SDC-Scissor will also use CARLA as a simulator. After the implementation of the pattern, I will close this issue unless there is something you want to point out before closing it. Okay?
